Sources/MapboxDirections/Intersection.swift

```diff
@@ -72,6 +84,48 @@ public struct Intersection {
      If road class information is unavailable, this property is set to `nil`.
      */
     public let outletRoadClasses: RoadClasses?
+
+    /**
+     The name of the tunnel that this intersection is a part of.
+
+     If this Intersection is not a tunnel entrance or exit, or if information is unavailable then this property is set to `nil`.
+     */
+    public let tunnelName: String?
+
+    /**
+     The type of toll collection mechanism.
+
+     If this Intersection is not a toll collection intersection, or if this information is unavailable then this property is set to `nil`.
+     */
+    public let tollCollection: TollCollection?
+
+    /**
+     The type of rest stop.
```

Comment:
RestStop itself contains a `type` property, so it seems like this `restStop` property is potentially a place to store more information about a rest stop?